webapp/.env.example

```diff
@@ -3,6 +3,9 @@
 REACT_APP_BACKEND_URI=https://localhost:40443/
 REACT_APP_AAD_AUTHORITY=https://login.microsoftonline.com/common
 REACT_APP_AAD_CLIENT_ID=
+# Authorization scopes to access webapi when using Azure AD authorization
+# example: api://12341234-1234-1234-1234-123412341234/access_as_user
+REACT_APP_AAD_API_SCOPE=
```

Comment:
move this below to the "Optional Variables" section?

Reply:
Yep. The optional variables is a better place for the key. Thanks for the hint.